cerealize/test.cc

```diff
@@ -143,9 +143,16 @@ TEST(Cerealize, BinarySerializesAndParses) {
   EventAppSuspend b;
   EventAppResume c;
 
-  CerealFileAppender(CurrentTestTempFileName()) << a << b << c;
+  CerealFileAppender<CerealFormat::Binary> f1(CurrentTestTempFileName());
+  f1 << a << b;
+  EXPECT_EQ(2u, f1.EntriesWritten());
+  EXPECT_EQ(115u, f1.BytesWritten());
+  CerealFileAppender<CerealFormat::Binary> f2(CurrentTestTempFileName());
```

Comment:
This test confuses me. Shouldn't `f2` fail at construction time since `f1` is still in scope and it holds the file?

Perhaps we need to make sure the file is accessed exclusively?

Or do modern filesystems not have this feature? In this case, I'd vote for serializing only `std::string`-s and checking `tellp()` before and after, otherwise it's too scary.